test/presubmit-tests.sh

```diff
@@ -1,4 +1,36 @@
 #!/usr/bin/env bash
 
-# TODO(approvers): replace with real tests
-true
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs the presubmit tests; it is started by prow for each PR.
+# For convenience, it can also be executed manually.
+# Running the script without parameters, or with the --all-tests
+# flag, causes all tests to be executed, in the right order.
+# Use the flags --build-tests, --unit-tests and --integration-tests
+# to run a specific set of tests.
+
+export GO111MODULE=on
+
+# Markdown linting failures don't show up properly in Gubernator resulting
+# in a net-negative contributor experience.
+export DISABLE_MD_LINTING=1
+
+source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/presubmit-tests.sh
+
+# TODO(mattmoor): integration tests
```

Comment:
Create an issue and refer here?

Reply:
@tcnghia I noticed most of the other presubmit-test.sh files in other knative-sandbox repos had this todo here, and was thinking maybe it was left for a reason. We can just remove.